src/clientiface.h

```diff
@@ -453,7 +453,7 @@ class ClientInterface {
 	std::vector<u16> getClientIDs(ClientState min_state=CS_Active);
 
 	/* get list of client player names */
-	std::vector<std::string> getPlayerNames();
+	const std::vector<std::string> &getPlayerNames() const { return m_clients_names; }
```

Comment:
yes i agree, but with this mode we ensure we have the correct behaviour for constness